deploy_scenarios/sap_s4hana_distributed_ha/ansible_playbook.yml

```
@@ -611,7 +611,7 @@
 
         sap_ha_pacemaker_cluster_vip_resource_group_name: vipnwasascs
 
-        sap_ha_pacemaker_cluster_storage_definition: "{{ vars['sap_vm_provision_' + sap_vm_provision_iac_platform + '_host_specifications_dictionary'][sap_vm_provision_host_specification_plan][ansible_hostname].storage_definition | list }}"
+        sap_ha_pacemaker_cluster_storage_definition: "{{ lookup('ansible.builtin.vars', 'sap_vm_provision_' + sap_vm_provision_iac_platform + '_host_specifications_dictionary')[sap_vm_provision_host_specification_plan][ansible_hostname].storage_definition | list }}"
```

did you try it? I think you should remove the ansible.builtin from it:
```suggestion
        sap_ha_pacemaker_cluster_storage_definition: "{{ lookup('vars', 'sap_vm_provision_' + sap_vm_provision_iac_platform + '_host_specifications_dictionary')[sap_vm_provision_host_specification_plan][ansible_hostname].storage_definition | list }}"
```

---

I've check it and it have the same results!

---

It is recommended to use the FQCN.
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/vars_lookup.html